src/js/views/forms/form/form.scss

```diff
@@ -15,10 +15,17 @@
   width: 100%;
 }
 
+.form__crumbs-title-section {
```

Comment:
there's an `overflow--hidden` class that'd probably work here.   One of a few that should probably be prefixed with `u-`  but none-the-less probably a good use case here.

However why does this div need a hidden overflow?